File: Python/Pelukis.py
from Seniman import Seniman
from Material import Material

class Pelukis(Seniman):
    def __init__(self, nama="", umur="", sejak="", nama_lukisan="", tahun_lukisan="", material=None):
        super().__init__(nama, umur, sejak)
        self.nama_lukisan = nama_lukisan
        self.tahun_lukisan = tahun_lukisan
        self.material = material if material else Material()

    def set_nama_lukisan(self, nama):
        self.nama_lukisan = nama

    def get_nama_lukisan(self):
        return self.nama_lukisan

    def set_tahun_lukisan(self, tahun):
        self.tahun_lukisan = tahun

    def get_tahun_lukisan(self):
        return self.tahun_lukisan

    def get_material(self):
        return self.material

    def add_data_lukisan(self, nama, umur, sejak, nama_lukisan, tahun_lukisan, material):
        super().add_data_seniman(nama, umur, sejak)
        self.nama_lukisan = nama_lukisan
        self.tahun_lukisan = tahun_lukisan
        self.material = material

File: javanies/Pelukis.java
public class Pelukis extends Seniman {
    private String namaLukisan;
    private String tahunLukisan;
    private Material material;

    public Pelukis() {}

    public Pelukis(String nama, String umur, String sejak,
                   String namaLukisan, String tahunLukisan, Material m) {
        super(nama, umur, sejak);
        this.namaLukisan = namaLukisan;
        this.tahunLukisan = tahunLukisan;
        this.material = m;
    }

    public void setNamaLukisan(String namaLukisan) { this.namaLukisan = namaLukisan; }
    public String getNamaLukisan() { return namaLukisan; }

    public void setTahunLukisan(String tahunLukisan) { this.tahunLukisan = tahunLukisan; }
    public String getTahunLukisan() { return tahunLukisan; }

    public Material getMaterial() { return material; }

    public void addDataLukisan(String nama, String umur, String sejak,
                               String namaLukisan, String tahunLukisan, Material m) {
        super.addDataSeniman(nama, umur, sejak);
        this.namaLukisan = namaLukisan;
        this.tahunLukisan = tahunLukisan;
        this.material = m;
    }
}
